src/modules/Department/index.tsx

```diff
@@ -6,21 +6,35 @@ import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
-import { Box, TextField } from '@mui/material';
+import { Box, TextField, Typography } from '@mui/material';
+
+import { useCreateDepartment } from './department.hook';
+import { useDepartMentList } from 'modules/details/details.hook';
+import { Button } from 'modules/shared/Button';
+import Loader from 'modules/Auth/components/Loader';
 
 function createData(name: string, calories: number) {
   return { name, calories };
 }
 export const DepartMent = () => {
   // todo
   const [department, setDepartment] = React.useState<string>('');
+
   const rows = [
     createData('Frozen yoghurt', 159),
     createData('Ice cream sandwich', 237),
     createData('Eclair', 262),
     createData('Cupcake', 305),
     createData('Gingerbread', 356),
   ];
+  const { data, isLoading } = useDepartMentList();
+  const { mutate, isLoading: isLoadingpatch } = useCreateDepartment();
```

Comment:
isLoadingpatch can be renamed to more specific state name as isCreatingDepartment